### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by JimboJoe** [comment]
!testme

**Comment by JimboJoe** [comment]
Tried installing a fresh instance of the current version, and to run the upgrade --> that failed: https://paste.yunohost.org/raw/irekevojug

File: scripts/upgrade [hunk 1]
```diff
@@ -26,11 +26,15 @@ db_user=$db_name
 db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
 random_key=$(ynh_app_setting_get --app=$app --key=random_key)
 email=$(ynh_user_get_info --username=$admin --key=mail)
+phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
```
Comment (JimboJoe) [inline scripts/upgrade:29]:
```suggestion
phpversion=$YNH_PHP_VERSION
```
wouldn't that cover the PHP migration...? @ericgaspar

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**